Python/Product/PythonTools/PythonTools/Project/PythonProjectLaunchProperties.cs

```diff
@@ -305,6 +305,9 @@ IProjectLaunchProperties properties
             IServiceProvider provider = null
         ) {
             var pathEnv = project.GetProjectAnalyzer().InterpreterFactory.Configuration.PathEnvironmentVariable;
+            if (pathEnv == null || pathEnv == String.Empty)

```

Comment:
`string.IsNullOrEmpty` is the idiomatic C# way to do this check. Also, can you add the curly braces around the body so that it matches the coding style of the rest of the file?